src/openzaak/selectielijst/tests/__init__.py

```diff
@@ -26,11 +28,22 @@ def mock_selectielijst_oas_get(m: Mocker) -> None:
     )
 
 
-def mock_resource_list(m: Mocker, resource: str) -> None:
-    url = f"{_get_base_url()}{resource}"
-    file = MOCK_FILES_DIR / f"{resource}.json"
-    with open(file, "rb") as response_data:
-        m.get(url, content=response_data.read())
+def mock_resource_list(
+    m: Mocker, resource: str, query_map: Optional[dict] = None
+) -> None:
```

Comment:
some docstring or more detailed type hint for `query_map` would be appreciated, for me it was not clear that it maps filenames with the dicts of the query params